src/ucp/wireup/wireup_cm.c

```diff
@@ -665,16 +669,20 @@ ucp_ep_cm_server_create_connected(ucp_worker_h worker, unsigned ep_init_flags,
                                   ucp_conn_request_h conn_request,
                                   ucp_ep_h *ep_p)
 {
+    uint64_t tl_bitmap = ucp_context_dev_tl_bitmap(worker->context,
+                                                   conn_request->dev_name);
     ucp_ep_h ep;
     ucs_status_t status;
 
     /* Create and connect TL part */
-    status = ucp_ep_create_to_worker_addr(worker, remote_addr, ep_init_flags,
+    status = ucp_ep_create_to_worker_addr(worker, tl_bitmap, remote_addr,
+                                          ep_init_flags,
                                           "conn_request on uct_listener", &ep);
     if (status != UCS_OK) {
         return status;
     }
 
+    ucs_assert(!(ucp_ep_get_tl_bitmap(ep) & ~tl_bitmap));
```

Comment:
maybe we should check before using it (i.e. before `ucp_ep_create_to_worker_addr()`)?

Reply:
no way, ep config key is not initialized before `ucp_ep_create_to_worker_addr`

Reply:
> no way, ep config key is not initialized before `ucp_ep_create_to_worker_addr`

I see, thanks!

Reply:
maybe move it to `ucp_ep_create_to_worker_addr()` then to cover other places where it's called?

Reply:
agree with @dmitrygx this is general assertion on the correctness of ucp_ep_create_to_worker_addr

Reply:
ok